fs/xfstests.py.data/ext4/64k_adv.yaml

```diff
@@ -0,0 +1,17 @@
+scratch_mnt: '/mnt/scratch'
+test_mnt: '/mnt/test'
+disk_mnt: '/mnt/loop-device'
+
+loop_type: !mux
+    type: 'loop'
+    loop_size: '9GiB'
+    # Option to provide disk for loop device creation,
+    # Uses '/' by default for file creation
+    disk: "null"
+
+fs_type: !mux
+    fs_ext4_64k_adv:
+        fs: 'ext4'
+        args: '-R xunit -e ext4/048 -L 10 -g quick'
+        mkfs_opt: '-b 65536 -O quota,inline_data,fast_commit'
```

Comment:
bigalloc feature is not supported on suse even after enabling allow_unsupported getting this error. 
IMO it will be good to have separate yaml for bigalloc to run where ever supported.
```
# cat /sys/module/btrfs/parameters/allow_unsupported
Y

[86820.288883] EXT4-fs (loop0): Couldn't mount RDWR because of SUSE-unsupported optional feature BIGALLOC.  Load module with allow_unsupported=1.
[86820.291897] EXT4-fs (loop0): Couldn't mount RDWR because of SUSE-unsupported optional feature BIGALLOC.  Load module with allow_unsupported=1.
```